hs_swat_modelinstance/models.py

```diff
@@ -46,12 +46,18 @@ def get_swat_model_objectives(self):
 
     @classmethod
     def _add_swat_objective(cls, model_objective, objectives):
```

Comment:
@mohamedmorsy OK now I see that you have implemented that check for the swat_model_instance. What about mudflow_instance?. The ```_validate_params()``` need to do similar check.